tests/integration/tests/test_cleanup.py

```diff
@@ -27,11 +29,64 @@ def test_node_cleanup(instances: List[harness.Instance], tmp_path):
     util.remove_k8s_snap(instance)
 
     # Check that the containerd-related folders are removed on snap removal.
+    all_paths = CONTAINERD_PATHS + [CNI_PATH]
     process = instance.exec(
-        ["ls", *CONTAINERD_PATHS], capture_output=True, text=True, check=False
+        ["ls", *all_paths], capture_output=True, text=True, check=False
     )
-    for path in CONTAINERD_PATHS:
+    for path in all_paths:
         assert f"cannot access '{path}': No such file or directory" in process.stderr
 
     util.setup_k8s_snap(instance, tmp_path)
     instance.exec(["k8s", "bootstrap"])
+
+
+@pytest.mark.node_count(2)
+@pytest.mark.disable_k8s_bootstrapping()
+def test_node_cleanup_new_containerd_path(instances: List[harness.Instance]):
```

Comment:
Please add a test level tag. That is the reason why the CI fails.
I think it is sufficient to run this `nightly`.
```
@pytest.mark.tags(tags.NIGHTLY)
```

Reply:
Done